### PR TITLE
new pull request

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -492,7 +494,7 @@
       <td>
       <div class="container">
         <h2>Fatima Batool</h2>
-        <p class="title">PhD student or Enrichment student</p>
+        <p class="title">PhD student or Enrichment student</p>f
```
Comment (KirstieJane) [inline index.html]: I think this "f" at the end of the line is just a typo, please delete that letter and then your pull request is ready to go!

**Review by KirstieJane — CHANGES_REQUESTED** [review]
This looks great @LinaGer94 - you have a tiny typo that I've flagged in another comment, but I'm going to approve this pull request so you can click merge when you've fixed that one!

**Review by KirstieJane — APPROVED** [review]

**Comment by KirstieJane** [comment]
@LinaGer94 - woooo - looks great! Merged!



You can see your information now at https://alan-turing-institute.github.io/2017-phd-students

**Comment by ghost** [comment]
hahahaha ! Thanks Kristie :) :)

**Pull request merged**